Runtimes/Unity/BepInEx.Unity.IL2CPP/Il2CppInteropManager.cs

```diff
@@ -249,6 +263,7 @@ private static void DownloadUnityAssemblies()
             using var zipStream = httpClient.GetStreamAsync(source).GetAwaiter().GetResult();
             using var zipArchive = new ZipArchive(zipStream, ZipArchiveMode.Read);
 
+            File.WriteAllText(LatestUnityVersionPath, VERSION);
```

Comment:
This should be written after all libraries successfully finish downloading and extracting.